src/saveload/marker.rs

```diff
@@ -35,7 +61,7 @@ impl<'a> EntityBuilder<'a> {
     /// ## Panics
     ///
     /// Panics in case there's no allocator added to the `World`.
-    pub fn marked<M>(self) -> Self
+    fn marked<M>(self) -> Self
```

Comment:
This still has duplicated docs

Reply:
Should I be removing the docs from the trait or the implementation? Do the docs from traits fall through to the implementations?

Reply:
Docs on the implementation of a trait only make sense if there's something special about the implementation. So for the lazy stuff it does make sense to document the delay of the insertions.